index.html

```diff
@@ -0,0 +1,127 @@
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </div>
+        <div class="nav-lists">
+            <a href="about.html" class="nav-link">About</a>
+            <a href="" class="nav-link">Schedule</a>
+            <a href="" class="nav-link">Contact-Us</a>
+            <button>Get Ticket</button>
+
+        </div>
+    </header>
+
+    <!-- first contents -->
+    <section class="intro-container">
+        <h1 class="intro-container-hea">Hello! Sharing World</h1>
+        <h2 class="intro-heading">The Best Experience Of <span class="intro-heading-span">Music</span> In 2022</h2>
+        <p>Music is life, music is love, music is everything.here you find list of all the upcoming concerts,global
+            tours and be able to buy a ticket</p>
+        <img src="./items/remaining-time-white.png" alt="the days remaining">
+
+    </section>
+    <Section class="explore-category">
+        <div class="explore-category-line">
+            <h2 class="explore-category-heading">Explore By Category</h2>
+            <hr>
+        </div>
+        <div class="category-content-wrap">
+            <div class="category-content">
+                <img src="./items/Dj Mixer.png" alt="pop music logo">
+                <h3 class="category-title">Pop</h3>
+                <p class="category-paragraph">Pop music is a genre of popular music that originated in its modern form
+                    during the mid-1950s in the United States.</p>
+            </div>
+            <div class="category-content">
+                <img src="./items/Bagpipes.png" alt="folk music logo">
+                <h3 class="category-title">Folk</h3>
+                <p class="category-paragraph">Folk music is a music genre that includes traditional folk music and the
+                    contemporary genre that evolved from the former.</p>
+            </div>
+            <div class="category-content">
+                <img src="./items/Saxophone.png" alt="blues music logo">
+                <h3 class="category-title">Blues</h3>
+                <p class="category-paragraph">An instrumental is a recording normally without any vocals, although it
+                    might include some inarticulate vocals.</p>
+            </div>
+            <div class="category-content">
+                <img src="./items/Guitar.png" alt="Rock logo">
+                <h3 class="category-title">Rock</h3>
+                <p class="category-paragraph">Rock music, also called rock and roll, rock & roll, or rock 'n' roll, is a
+                    form of popular music that emerged in the 1950s.</p>
+            </div>
+            <div class="category-content">
+                <img src="./items/Trumpet.png" alt="Jazz logo">
+                <h3 class="category-title">Jazz</h3>
+                <p class="category-paragraph">People who enjoy jazz, blues, or soul music were found to be more
+                    extroverted with high self-esteem.</p>
+            </div>
```

Comment:
- [OPTIONAL] This is marked optional, because it was not mentioned by the previous reviewers. It is highly recommended you implement this in your project please. Your present layout is not what is expected on the mobile screen size for this section.
- Below is how your layout for this section supposed to be.
The middle section individual articles have their elements organized in an horizontal row, with the image at the left, title in the center, and description at the right.
-  Kindly look into it and make this section look like that of guiding template.

Take a look at the both screenshots :point_down: 

### Your Design

![your-main-program-section](https://user-images.githubusercontent.com/26597432/197170829-b63e47f6-116d-41f4-985c-1697ebb192ff.png)


### Guiding Template

![mobile-main-program-section](https://user-images.githubusercontent.com/26597432/197170875-cc0e2206-24d7-42af-9525-2275c0df7424.png)